app/views/hub/messages/index.html.erb

```diff
@@ -11,12 +11,15 @@
   <% if @messages_by_day.present? %>
     <ul class="message-list" aria-live="polite" data-js="messages-pub-sub" data-client-id="<%=@client.id%>">
 
-      <% @messages_by_day.each do |datetime, contact_records| %>
-
+      <% @messages_by_day.each_with_index do |(datetime, contact_records), day_index| %>
```

Comment:
I'd never used this syntax before, but encountered it here and thought it worked okay: https://stackoverflow.com/a/2083597/399726